src/lib.rs

```diff
@@ -2948,8 +2948,8 @@ impl Build {
     /// Returns compiler path, optional modifier name from whitelist, and arguments vec
     fn env_tool(&self, name: &str) -> Option<(PathBuf, Option<String>, Vec<String>)> {
         let tool = match self.getenv_with_target_prefixes(name) {
-            Ok(tool) => tool,
-            Err(_) => return None,
+            Ok(tool) if !tool.trim().is_empty() => tool,
```

Comment:
```suggestion
            Ok(tool) if !tool.trim().is_empty() => tool.trim(),
```

Reply:
trim before is_empty was used for avoid  empty value(such the value is space). maybe we should combine two

Reply:
Yeah that makes sense